content/en/blog/2022/opamp/index.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Using OpenTelemetry OpAMP to modify service telemetry on the go
+linkTitle: OpAMP service usage
+date: 2022-10-12
+author: >-
+  [Nimrod Shlagman](https://github.com/nemoshlag) (Cisco)
+---
+
+How verbose should your service telemetry be? Should a service output all
+traces, metrics, and logs 100% of the time? How much of the service traffic
+should be sampled? I would like to suggest the answer of “it depends”. Desired
+telemetry data differs in a service lifecycle from development to continuous
+deployment. It changes when a client faces an error or our service has been
+thrown into major scale. In an ideal world, we would modify the entire service’s
+telemetry without the need for code changes or deployment. Thanks to Otel
+[OpAMP](https://github.com/open-telemetry/opamp-go) service and the people
+behind it, I believe the time has arrived.
+
+OpenTelemtry, also known as Otel, is a vendor-neutral open-source observability
```

Comment:
you don't need that on the otel blog.

Reply:
Note: line 19 - OpenTelemetry is misspelled.

@cartermp, @svrnm, @austinlparker and @chalin: I was wondering if we've standardized on how we spell/use 'open source'? Is it "open source", "open-source", "Open Source" etc.   
(this has come up in another, ie work related, context and was the subject of much debate!)

Reply:
I believe "OTel" (rather than "Otel") is correct.

Reply:
> Note: line 19 - OpenTelemetry is misspelled.
> 
> @cartermp, @svrnm, @austinlparker and @chalin: I was wondering if we've standardized on how we spell/use 'open source'? Is it "open source", "open-source", "Open Source" etc. (this has come up in another, ie work related, context and was the subject of much debate!)

is there something from the LF/CNCF on that? We stick with that.